cmd/version.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/dapr/cli/pkg/print"
+)
+
+var output string
+
+var VersionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print Dapr Runtime and Cli's version.",
+	Example: `
+# Version for Dapr
+dapr version --output json
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if output != "" && output != "json" {
```

Comment:
Should be easier to read with a switch case.
```go
switch output {
case "":
    // normal output
case "json":
    // json output
default:
    // fail and exit
}
```

Reply:
hi, @shubham1172 now, there are only two cases for output, one is json, another is normal, that  is equal to default. Do you think we need to change this style in codes?

Reply:
done.

Reply:
We do not need the first if condition anymore, right?
`print.FailureStatusEvent(os.Stdout, "An invalid output format was specified.")` can be a part of the `default` case.